resources/application-connector/charts/application-broker/templates/cluster-role.yaml

```diff
@@ -29,4 +30,11 @@ rules:
 - apiGroups: ["servicecatalog.k8s.io"]
   resources: ["servicebrokers"]
   verbs: ["get", "create", "delete", "list", "update"]
-
+# subscriptions are created by the application-broker to enable events flowing from a channel to the broker
+- apiGroups: ["messaging.knative.dev"]
+  resources: ["subscriptions"]
+  verbs: ["get", "create", "delete", "list", "update"]
```

Comment:
`get` is not used in the broker if I'm not mistaken.

Reply:
Fixed